linux/utils/add_local_dvd_repo.yml

```diff
@@ -80,23 +80,22 @@
 - name: "Print CDROM device mount fact"
   ansible.builtin.debug: var=guest_cdrom_mount_fact
 
-- name: "Umount CDROM device if it has been mounted"
-  ansible.posix.mount:
-    path: "{{ item.mount }}"
-    state: absent
-  delegate_to: "{{ vm_guest_ip }}"
+- name: "Umount auto mounted CDROM device"
+  include_tasks: set_mount_point.yml
+  vars:
+    mount_point_path: "{{ item.mount }}"
+    mount_point_src: "{{ item.device }}"
+    mount_state: "unmounted"
   with_items: "{{ guest_cdrom_mount_fact }}"
   when: guest_cdrom_mount_fact | length >= 1
 
 - name: "Mount CDROM device to {{ guest_cdrom_mount_path }}"
-  ansible.posix.mount:
-    path: "{{ guest_cdrom_mount_path }}"
-    src: "/dev/{{ guest_cdrom_device_name }}"
-    fstype: iso9660
-    state: ephemeral
-  register: guest_cdrom_mount_result
-  become: true
```

Comment:
Is this `become: true` still needed? I saw this is used in 2 places.

Reply:
Not needed any more as we require root privileges in our [prerequisites](https://github.com/vmware/ansible-vsphere-gos-validation?tab=readme-ov-file#prerequisites)